src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/__tests__/CreateAntigenTestProfileViewModelTests.swift

```diff
@@ -5,13 +5,24 @@
 import XCTest
 @testable import ENA
 
-/* EXPOSUREAPP-12801
 class CreateAntigenTestProfileViewModelTests: CWATestCase {
 	
 	func testGIVEN_EmptyAntigenTestProfile_THEN_SetValues() {
```

Comment:
The name somehow doesn't fit anymore?
Is this case with empty profiles tested somewhere else?

Reply:
Good catch, fixed.